package.json

```diff
@@ -80,8 +80,10 @@
     "prettier": "^2.8.7",
     "prettier-plugin-organize-imports": "^3.2.2",
     "rimraf": "^5.0.0",
-    "semver": "^7.4.0",
-    "typescript": "^5.0.3"
+    "typescript": "^5.0.3",
+    "zod": "^3.21.4",
+    "zod-validation-error": "^1.3.0",
```

Comment:
these should both be in `dependencies`

Reply:
Yeah, I wasn't sure about that. What logic do we choose to follow when deciding between dev deps/deps?

Reply:
Our dev deps won't be installed when someone adds this package to their own npm project, or installs it globally.

Reply:
Got it. Thanks!